tests/queryset_pickle/models.py

```diff
@@ -58,3 +58,20 @@ class SomeModel(models.Model):
 
 class M2MModel(models.Model):
     groups = models.ManyToManyField(Group)
+
+
+class AbstractEvent(Event):
+    class Meta:
+        abstract = True
+        ordering = ['title']
+
+
+class MyEvent(AbstractEvent):
+    pass
+
+
+class ConcreteEventFK(models.Model):
+    event = models.ForeignKey('MyEvent', related_name='concrete_events', on_delete=models.CASCADE)
```

Comment:
`related_name` is not related with this issue, so I think we can drop it and use a generic name.